firebase-app.html

```diff
@@ -69,22 +69,34 @@
           },
 
           /**
+           * The URL of your Firebase Storage Bucket. You can find this
+           * URL in the Database panel of the Firebase Console.
+           * Available on the Firebase Console.
+           *
+           * For example: 'gs://polymerfire-test.appspot.com/'
```

Comment:
Does the URL need to start with `gs://`? In the demo, the `storage-url` value is simply "polymerfire-test.appspot.com".